src/server/components/dashboard/listRouter.ts

```diff
@@ -59,6 +65,8 @@ listRouter.post("/:listId/annual-review-date", annualReview.editDatePostControll
 listRouter.get("/:listId", listsEditController);
 listRouter.post("/:listId", listsEditPostController);
 
+listRouter.get("/:listId/csv-export", validateActionPermission, listsExportController);
```

Comment:
It would just be `/:listId/csv` 

in terms of RESTful apis - that translates to "get the CSV for listId"